samcli/cli/core/command.py

```diff
@@ -40,7 +40,9 @@ def _format_options(
         params: List[Parameter],
         formatter: RootCommandHelpTextFormatter,
         formatting_options: Dict[str, Dict],
+        write_rd_overrides: Optional[Dict[str, Any]] = None,
     ):
+        write_rd_overrides = write_rd_overrides if write_rd_overrides else {}
```

Comment:
nit: simpler version;
```suggestion
        write_rd_overrides = write_rd_overrides or {}
```